### PR TITLE
Allow to turn off GH issue creation but keep on as default

### PR DESCRIPTION
**Pull request by AlexDHoffer**

### CODE REVIEW

**Comment by kingthorin** [comment]
To address the DCO requirement you'll need to sign-off the commit(s):

- https://github.com/zaproxy/zaproxy/blob/develop/CONTRIBUTING.md#developer-certificate-of-origin
- https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---signoff

**Comment by kingthorin** [comment]
Thanks for tackling this!

**Comment by AlexDHoffer** [comment]
I force-pushed to resolve sign-off requirement

**Comment by kingthorin** [comment]
Great, thanks!

File: packages/scans/src/index.js
Comment (sshniro) [inline packages/scans/src/index.js:45]: It's better to upload the artifacts and exit the function after LOC 12, as the further down the files, it will try to generate the difference between the current run and the previous run.

Therefore:
```
LOC: 12
if (allowIssueWriting) {
  actionHelper.uploadArtifacts(workSpace, mdReportName, jsonReportName, htmlReportName);
  return;
}

....
let repo;
let tmp = repoName.split('/');
owner = tmp[0];
repo = tmp[1];

```

Comment (AlexDHoffer) [reply packages/scans/src/index.js:45]: Sure, I can push up a commit in a sec that does it this way.

**Review by sshniro — APPROVED** [review]

File: packages/scans/src/index.js
Comment (thc202) [inline packages/scans/src/index.js:14]: Was this meant to be negated?

Comment (AlexDHoffer) [reply packages/scans/src/index.js:14]: Not sure what you mean by negated? The other commenter pointed out there is a less verbose way to achieve this result, so we're adjusting to use that. I don't see anything being erased in this PR, but maybe I am hard of sight today!

Comment (sshniro) [reply packages/scans/src/index.js:14]: yes, the logic should be inverse, my bad.

Comment (AlexDHoffer) [reply packages/scans/src/index.js:14]: Oh! Lol. Sorry, I didn't even notice. My eyes just glanced over it. I will change that.

Comment (AlexDHoffer) [reply packages/scans/src/index.js]: @thc202 This was resolved, are you comfortable approving now?

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Comment by AlexDHoffer** [comment]
> Thank you!

You got it, I will have a PR up today as well that will implement this option for the baseline scan Action

**Comment by thc202** [comment]
Great! We do need to release this one first though (well, at least to merge it).

**Comment by AlexDHoffer** [comment]
@thc202 The companion piece to this pull request is here: https://github.com/zaproxy/action-baseline/pull/58 

Let me know if I can assist somehow in "releasing this one first" (I assume that is all zaproxy side)!

**Comment by thc202** [comment]
For the record, the library was already released and the referenced pull request merged.

**Pull request merged**
